tests/benchdnn/inputs/graph/op/f32/gelu.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Shall we utilize the op_kind rewriter?

Reply:
The attribute is new and not contained in the original unary json file. I tried to see if `--op-attr` can work if the input file does not contain the attribute, but it failed. So I added a new test file here.